Assets/MRTK/Core/Definitions/Devices/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -43,28 +43,30 @@ public SystemType DefaultControllerVisualizationType
 
         [SerializeField]
         [Tooltip("Check to obtain controller models from the platform SDK. If left unchecked, the global models will be used. Note: this value is overridden by controller definitions.")]
-        private bool useDefaultModels = false;
+        [FormerlySerializedAs("useDefaultModels")]
+        private bool usePlatformModels = false;
 
         /// <summary>
         /// Check to obtain controller models from the platform SDK. If left unchecked, the global models will be used. Note: this value is overridden by controller definitions.
         /// </summary>
         public bool UseDefaultModels
         {
-            get => useDefaultModels;
-            private set => useDefaultModels = value;
+            get => usePlatformModels;
+            private set => usePlatformModels = value;
         }
 
         [SerializeField]
         [Tooltip("The default controller model material when loading platform SDK controller models. This value is used as a fallback if no controller definition exists with a custom material type.")]
-        private Material defaultControllerModelMaterial;
+        [FormerlySerializedAs("defaultControllerModelMaterial")]
+        private Material platformModelMaterial;
 
         /// <summary>
         /// The default controller model material when loading platform SDK controller models. This value is used as a fallback if no controller definition exists with a custom material type.
         /// </summary>
-        public Material DefaultControllerModelMaterial
+        public Material PlatformModelMaterial
```

Comment:
To prevent this from being a breaking change, we should keep the original property name around and obsolete it. Since the original had a private set, we can probably just include a getter:

```c#
/// <summary>
/// The default controller model material when loading platform SDK controller models. This value is used as a fallback if no controller definition exists with a custom material type.
/// </summary>
[Obsolete("Use PlatformModelMaterial instead.")]
public Material DefaultControllerModelMaterial => platformModelMaterial;